hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,9 @@
+RELEASE_TYPE: minor
+
+This release adds :func:`hypothesis.currently_in_test_context`, which can be used
+to check whether the calling code is currently running inside an
```

Comment:
an → a ?

(unless this is intended to go with a pronounced `@`?)

Reply:
(Ditto for the docstring.)

Reply:
I tend to write "an", mentally pronouncing "an at-given", but a quick grep through our docs reveals that we use both... and I'm going to leave that as a problem for another day 😅

Reply:
English! How does it even? 😄️